source/components/DatasetSelector.js

```diff
@@ -81,6 +86,7 @@ export default class DatasetSelector extends React.Component {
 DatasetSelector.propTypes = {
   labels: React.PropTypes.array,
   onDatasetSelected: React.PropTypes.func,
+  onCustomDataset: React.PropTypes.func,
```

Comment:
We may as well add `onCustomDataset` to `defaultProps`, too. I don't suppose there's a way to lint for that as well?